addons/website/views/snippets/s_striped.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_striped" name="Striped section">
+    <section class="s_striped o_cc o_cc5 pt56 pb96" data-oe-shape-data="{'shape':'web_editor/Origins/04_001','colors':{'c3': '#FFFFFF'},'flip':['y'], 'showOnMobile':true}">
```

Comment:
Missing `style="position: relative;"`
It is added to sections containing a shape by the editor ;)

Reply:
Thanks, didn't pay attention to that, might need to apply this as well to some of my others PR's 😅